phpseclib/Crypt/Base.php

```diff
@@ -471,6 +471,24 @@ class Crypt_Base
     var $openssl_options;
 
     /**
+     * Has the key length explicitly been set or should it be derived from the key, itself?
+     *
+     * @see setKeyLength()
```

Comment:
I assume `@see Base::setKeyLength()` is better than `@see Crypt_Base::setKeyLength()`? I say that based on your comment at https://github.com/phpseclib/phpseclib/pull/827#discussion_r40877854

And should I do `@see Base::setKeyLength()` or `@see \phpseclib\Crypt\Base::setKeyLength()`? phpseclib does both throughout it's codebase. We ought to be using one consistent standard.